gadget-container/entrypoint.sh

```diff
@@ -215,29 +215,6 @@ else
   fi
 fi
 
-# Choose what kind of tools based on the configuration detected
-TOOLS_MODE="$INSPEKTOR_GADGET_OPTION_TOOLS_MODE"
-
-if [ "$TOOLS_MODE" = "auto" ] || [ -z "$TOOLS_MODE" ] ; then
-  if test -f /sys/kernel/btf/vmlinux; then
-    echo "BTF is available at /sys/kernel/btf/vmlinux: Using CO-RE based tools"
-    TOOLS_MODE="core"
-  elif test -f /boot/vmlinux-$KERNEL; then
-    echo "BTF is available at /boot/vmlinux-$KERNEL: Using CO-RE based tools"
-    TOOLS_MODE="core"
-  else
-    echo "vmlinux not found. Using standard tools"
-    TOOLS_MODE="standard"
-  fi
-fi
-
-# Create symlinks for tools according to the value of TOOLS_MODE
-if [ "$TOOLS_MODE" = "core" ] ; then
-  ln -s /bin/libbpf-tools/ /bin/gadgets
-elif [ "$TOOLS_MODE" = "standard" ] ; then
-  ln -s /usr/share/bcc/tools/ /bin/gadgets
```

Comment:
Since `/bin/gadgets` is no longer created, the following comment should be updated:

https://github.com/kinvolk/inspektor-gadget/blob/40376f7173cede226eb132f6b9ffd745c74def93/cmd/kubectl-gadget/bcck8s.go#L35-L37

Reply:
I removed that command as all tools present in bcck8s.go use `/usr/share/bcc/tools`.